web_scraper.py

```diff
@@ -0,0 +1,121 @@
+# install selenium and chromedriver
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.chrome.options import Options
+import math
+import pandas as pd
+import os
+import time
+
+
+def setup_chrome():
+    chrome_options = Options()
+    chrome_options.add_argument("--headless")
+    chrome_options.add_argument("--window-size=1920,1080")
+    chrome_options.add_argument("--disable-gpu")
+    chrome_options.add_argument("--disable-extensions")
+    chrome_options.add_experimental_option("useAutomationExtension", False)
+    chrome_options.add_argument("--proxy-server='direct://'")
+    chrome_options.add_argument("--proxy-bypass-list=*")
+    chrome_options.add_argument("--start-maximized")
+    chrome_options.add_argument("--headless")
```

Comment:
Great! I like that you're making it headless! Did this argument need to be added twice?